demisto_sdk/commands/format/tests/test_formatting_json_test.py

```diff
@@ -755,3 +755,20 @@ def test_json_run_format_exception_handling(self, format_object, mocker, capsys)
         formatter.run_format()
         stdout, _ = capsys.readouterr()
         assert 'Failed to update file my_file_path. Error: MY ERROR' in stdout
+
+    def test_set_fromversion_six_new_contributor_pack(self, pack, mocker):
+        """
+        Given
+            - An integration from new countributed pack
+        When
+            - Run format command
+        Then
+            - Ensure that the integration fromversion is set to 6.0.0
+        """
+        pack.pack_metadata = {'support': 'partner',
+                              'currentVersion': '1.0.0'}
+        incident_type = pack.create_incident_type(name='TestType', content={'fromVersion': '5.5.0'})
```

Comment:
In order for us to be able to test all entities (not only incidents) let's add parameterized decorator to this function and give a different JSON entity every time (indicator_type, incident_fields, etc...).